test/helpers/kubectl.go

```diff
@@ -1828,7 +1828,7 @@ iteratePods:
 		if err != nil || ep.Identity == nil || ep.Identity.ID == 0 {
 			ginkgoext.By("Restarting unmanaged pod %s/%s", namespace, pod.Name)
 			cmd = KubectlCmd + " -n " + namespace + " delete pod " + pod.Name
-			res = kub.ExecShort(cmd)
+			res = kub.Exec(cmd)
```

Comment:
If we want to increase timeout here, let's use `ExecMiddle`, which bumps 10s to 30s instead of 4 minutes (shouldn't 30s be enough for a pod to terminate ?).

Reply:
Some pods are quite slow to terminate, we don't control the pods, I'd rather have a conservative value for this. The thing is, the closer you get to a chance of timeout triggering, the great the chance of tests failing. If we weren't using `kubectl` here, we wouldn't have to worry all that much, we could just watch for it to get deleted.